src/test/java/edu/unc/lib/boxc/migration/cdm/services/SipServiceTest.java

```diff
@@ -1025,6 +1025,352 @@ public void generateSipsWithDefaultPermissions() throws Exception {
+        groupOptions.setGroupField("groupa");
+        GroupMappingService groupService = testHelper.getGroupMappingService();
+        groupService.generateMapping(groupOptions);
+        groupService.syncMappings(makeDefaultSyncOptions());
+        // permissions must be set after the grouping
+        testHelper.generateWorkPermissionsMapping(UserRole.canViewMetadata, UserRole.canViewMetadata);
+
+        List<MigrationSip> sips = service.generateSips(makeOptions());
+        assertEquals(1, sips.size());
+        MigrationSip sip = sips.get(0);
+
+        assertTrue(Files.exists(sip.getSipPath()));
+
+        DepositDirectoryManager dirManager = testHelper.createDepositDirectoryManager(sip);
+
+        Model model = testHelper.getSipModel(sip);
+
+        Bag depBag = model.getBag(sip.getDepositPid().getRepositoryPath());
+        List<RDFNode> depBagChildren = depBag.iterator().toList();
+        assertEquals(4, depBagChildren.size());
+
+        Resource workResc1 = testHelper.getResourceByCreateTime(depBagChildren, "2005-11-23");
+        Bag work1Bag = model.getBag(workResc1);
+        testHelper.assertGroupedWorkPopulatedInSip(workResc1, dirManager, model, "grp:groupa:group1", false,
+                stagingLocs.get(0), stagingLocs.get(1));
+        // Assert that children of grouped work do not have permissions
+        Resource work1File1Resc = testHelper.findChildByStagingLocation(work1Bag, stagingLocs.get(0));
+        assertFalse(work1File1Resc.hasProperty(CdrAcl.canViewMetadata, PUBLIC_PRINC));
+        assertFalse(work1File1Resc.hasProperty(CdrAcl.canViewMetadata, AUTHENTICATED_PRINC));
+        Resource work1File2Resc = testHelper.findChildByStagingLocation(work1Bag, stagingLocs.get(1));
+        assertFalse(work1File2Resc.hasProperty(CdrAcl.canViewMetadata, PUBLIC_PRINC));
+        assertFalse(work1File2Resc.hasProperty(CdrAcl.canViewMetadata, AUTHENTICATED_PRINC));
+
+        // Assert that grouped worked have permissions
+        Resource workResc2 = testHelper.getResourceByCreateTime(depBagChildren, "2005-12-08");
+        testHelper.assertObjectPopulatedInSip(workResc2, dirManager, model, stagingLocs.get(2), null, "27");
+        assertTrue(workResc2.hasProperty(CdrAcl.canViewMetadata, PUBLIC_PRINC));
+        assertTrue(workResc2.hasProperty(CdrAcl.canViewMetadata, AUTHENTICATED_PRINC));
+        Resource workResc3 = testHelper.getResourceByCreateTime(depBagChildren, "2005-12-09");
+        assertTrue(workResc3.hasProperty(CdrAcl.canViewMetadata, PUBLIC_PRINC));
+        assertTrue(workResc3.hasProperty(CdrAcl.canViewMetadata, AUTHENTICATED_PRINC));
+        Resource workResc4 = testHelper.getResourceByCreateTime(depBagChildren, "2005-12-10");
+        assertTrue(workResc4.hasProperty(CdrAcl.canViewMetadata, PUBLIC_PRINC));
+        assertTrue(workResc4.hasProperty(CdrAcl.canViewMetadata, AUTHENTICATED_PRINC));
+
+        assertPersistedSipInfoMatches(sip);
+    }
```

Comment:
We should also test with regular single file works to make sure they set permissions on works but not files.